Client/Frontend/Strings.swift

```diff
@@ -4701,6 +4706,21 @@ extension String {
         tableName: "TabLocation",
         value: "Secure connection. Enhanced Tracking Protection is off.",
         comment: "Accessibility label for the security icon in url bar")
+    public static let TabLocationLockButtonLargeContentTitle = MZLocalizedString(
+        key: "TabLocation.LockButton.LargeContentTitle.v122",
+        tableName: nil,
+        value: "Tracking Protection",
+        comment: "Large content title for the lock button")
```

Comment:
This comment is used to provide a description for contributors who translate strings in [Pontoon](https://pontoon.mozilla.org/). I think we should give them a bit more context here as to what this string is used for i.e. for large content viewer accessibility feature on toolbar buttons.